geomstats/geometry/discrete_curves.py

```diff
@@ -423,10 +423,11 @@ def __init__(self, ambient_manifold, metric=None, translation_invariant=True):
                 )
         else:
             self.ambient_metric = metric
+        self.default_point_type = "matrix"
         self.l2_metric = L2CurvesMetric(ambient_manifold, metric)
         self.translation_invariant = translation_invariant
 
-    def srv_transform(self, curve):
+    def srv_transform(self, curve, tol=1e-10):
```

Comment:
Is `gs.atol` to restrictive to use as default?

Reply:
Done.